### PR TITLE
New: Support for UBL Tax Category Reason

### LINKED ISSUE #946: Add support for UBL Tax Category Reason
A customer suggested adding support for the UBL Tax Category reason, which is necessary for certain tax categories.

### PR DESCRIPTION
**Pull request by alexmigf**
closes #946

### CODE REVIEW

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

File: ubl/Settings/TaxesSettings.php [hunk 1]
```diff
@@ -96,27 +99,53 @@ public function output_table_for_tax_class( $slug, $name ) {
...
+				$options = $this->get_available_categories();
+				break;
+			case 'reason':
+				$options = self::get_available_reasons();
+				break;
+			default:
+				$options = array();
+		}
+
+		$select = '<select name="wpo_wcpdf_settings_ubl_taxes[' . $type . '][' . $id . '][reason]"><option value="">' . __( 'Default', 'woocommerce-pdf-invoices-packing-slips' ) . '</option>';
```
Comment (MohamadNateqi) [inline ubl/Settings/TaxesSettings.php:140]: The `reason` should be replaced with `$for`:
`wpo_wcpdf_settings_ubl_taxes[' . $type . '][' . $id . '][reason]`

**Review by MohamadNateqi — CHANGES_REQUESTED** [review]

File: ubl/Settings/TaxesSettings.php (hunk 1 above)
Comment (alexmigf) [reply ubl/Settings/TaxesSettings.php]: Good catch!

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

**Pull request merged**